recipe/meta.yaml

```
@@ -61,7 +61,9 @@ test:
     # Need these for AOT. Do not init msvc as it may not be present
     - {{ compiler('c') }}      # [not win]
     - {{ compiler('cxx') }}    # [not win]
-    - numpy >=1.15
+    # Working around an issue with NumPy 1.19
+    # xref: https://github.com/numba/numba/issues/5929
+    - numpy >=1.15,<1.19
```

xref: https://github.com/numba/numba/issues/5929

---

IIUC, ref. https://github.com/numba/numba/issues/5929#issuecomment-651109741, this is fixed in `1.19.1`, so we could do
```suggestion
    # Working around an issue with NumPy 1.19.0
    # xref: https://github.com/numba/numba/issues/5929
    - numpy >=1.15,!=1.19.0
```
Or did I miss something?

---

I don't know. Asked here ( https://github.com/numba/numba/issues/5929#issuecomment-675154168 ).

That said, this is just in the tests. So I don't think we need to worry about this too much. Hoping by the next Numba release we will be able to drop this.